### PR TITLE
Report 'Log' domain entries as 'console' events

### LINKED ISSUE #446: feat(Page): report 'Log' domain entries as 'console' events
* [feat(Page): report 'Log' domain entries as 'console' events](https://github.com/GoogleChrome/puppeteer/commit/e236887bbbd015e46fe036ef1caf957b52d70251) (#2400)

### PR DESCRIPTION
**Pull request by kblok**
closes #446

### CODE REVIEW

File: lib/PuppeteerSharp.Tests/PageTests/Events/ConsoleTests.cs [hunk 1]
```diff
@@ -101,5 +101,19 @@ void EventHandler(object sender, ConsoleEventArgs e)
 
             Assert.Equal("JSHandle@object", message.Text);
         }
+
+        [Fact]
+        public async Task ShouldTriggerCorrectLog()
+        {
+            await Page.GoToAsync("about:blank");
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/PageTests/Events/ConsoleTests.cs]: use `TestConstants.AboutBlank`

File: lib/PuppeteerSharp/Page.cs [hunk 2]
```diff
@@ -1606,7 +1607,30 @@ private decimal ConvertPrintParameterToInches(object parameter)
...
+        {
+            if (e.Entry.Args != null)
+            {
+                foreach (var arg in e.Entry?.Args)
+                {
+                    RemoteObjectHelper.ReleaseObject(Client, arg, _logger);
+                }
             }
+            Console?.Invoke(this, new ConsoleEventArgs(new ConsoleMessage(e.Entry.Level, e.Entry.Text)));
+            /*
```
Comment (Meir017) [inline lib/PuppeteerSharp/Page.cs]: delete js comments

Comment (kblok) [reply lib/PuppeteerSharp/Page.cs]: ups :joy:

**Comment by kblok** [comment]
Done @Meir017

**Pull request merged**